easybuild/easyconfigs/s/snakemake/snakemake-8.2.1-foss-2023a.eb

```diff
@@ -0,0 +1,144 @@
+    }),
+    ('ConfigArgParse', '1.7', {
+        'checksums': ['e7067471884de5478c58a511e529f0f9bd1c66bfef1dea90935438d6c23306d1'],
+    }),
+    ('argparse-dataclass', '2.0.0', {
+        'modulename': 'argparse_dataclass',
+        'source_tmpl': 'argparse_dataclass-%(version)s.tar.gz',
+        'checksums': ['09ab641c914a2f12882337b9c3e5086196dbf2ee6bf0ef67895c74002cc9297f'],
+    }),
+    ('snakemake-interface-common', '1.15.1', {
+        'modulename': 'snakemake_interface_common',
+        'source_tmpl': 'snakemake_interface_common-%(version)s.tar.gz',
+        'checksums': ['5a779c7a0f27412bce26e1dfe4ccc712d04e0c0b9ceaa1640335a35d02509732'],
+    }),
+    ('reretry', '0.11.8', {
+        'checksums': ['f2791fcebe512ea2f1d153a2874778523a8064860b591cd90afc21a8bed432e3'],
+    }),
+    ('snakemake-interface-storage-plugins', '3.0.0', {
+        'modulename': 'snakemake_interface_storage_plugins',
+        'source_tmpl': 'snakemake_interface_storage_plugins-%(version)s.tar.gz',
+        'checksums': ['f20d85ee7e86a1e2ffa3f72e2385dd5abb17fa7b58a26cba8ba59096872fe169'],
+    }),
+    ('snakemake-interface-executor-plugins', '8.2.0', {
+        'modulename': 'snakemake_interface_executor_plugins',
+        'source_tmpl': 'snakemake_interface_executor_plugins-%(version)s.tar.gz',
+        'checksums': ['4c74e3e1751bab6b266baf8688e854b8b4c5c5e10f5e34c581f42d69af4ff13b'],
+    }),
+    ('smart-open', '6.4.0', {
+        'sources': ['smart_open-%(version)s.tar.gz'],
+        'checksums': ['be3c92c246fbe80ebce8fbacb180494a481a77fcdcb7c1aadb2ea5b9c2bee8b9'],
+    }),
+    ('jupyter-core', '5.7.1', {
+        'modulename': 'jupyter_core',
+        'source_tmpl': 'jupyter_core-%(version)s.tar.gz',
+        'checksums': ['de61a9d7fc71240f688b2fb5ab659fbb56979458dc66a71decd098e03c79e218'],
+    }),
+    ('fastjsonschema', '2.19.1', {
+        'checksums': ['e3126a94bdc4623d3de4485f8d468a12f02a67921315ddc87836d6e456dc789d'],
+    }),
+    ('nbformat', '5.9.2', {
+        'source_tmpl': '%(name)s-%(version)s-py3-none-any.whl',
+        'checksums': ['1c5172d786a41b82bcfd0c23f9e6b6f072e8fb49c39250219e4acfff1efe89e9'],
+    }),
+    ('immutables', '0.20', {
+        'checksums': ['1d2f83e6a6a8455466cd97b9a90e2b4f7864648616dfa6b19d18f49badac3876'],
+    }),
+    ('humanfriendly', '10.0', {
+        'checksums': ['6b0b831ce8f15f7300721aa49829fc4e83921a9a301cc7f606be6686a2288ddc'],
+    }),
+    ('connection-pool', '0.0.3', {
+        'sources': ['connection_pool-%(version)s.tar.gz'],
+        'checksums': ['bf429e7aef65921c69b4ed48f3d48d3eac1383b05d2df91884705842d974d0dc'],
+    }),
+    ('conda-inject', '1.3.1', {
+        'sources': ['conda_inject-%(version)s.tar.gz'],
+        'checksums': ['9e8d902230261beba74083aae12c2c5a395e29b408469fefadc8aaf51ee441e5'],
+    }),
+    (name, version, {
+        'checksums': ['f177ada98188a10602aa2ed758fc9b7ff13e7107c5e582463ae681c525c28f0c'],
+    }),
+    # Also install some of the snakemake executors
```

Comment:
There are more executors, but I had issues with some of them and no way to test them anyway (and they pull in a lot more packages)